packages/salesforcedx-vscode-apex/src/languageUtils/languageClientUtils.ts

```diff
@@ -1,23 +1,30 @@
 /*
- * Copyright (c) 2017, salesforce.com, inc.
+ * Copyright (c) 2025, salesforce.com, inc.
```

Comment:
just a nit, we don't usually modify this for old files, but just add current year for new ones.